File: trunk/C#/Sample Project/SampleProject/SampleProject.UI/Base/BaseWebPart.cs
using System;
using System.ComponentModel;
using System.Web;
using System.Web.UI;
using Microsoft.SharePoint.WebControls;

namespace SampleProject.UI.WebParts
{
    public class BaseWebPart : Microsoft.SharePoint.WebPartPages.WebPart
    {
        #region WebPart Methods
        protected override void CreateChildControls()
        {
            base.CreateChildControls();
        }


        protected override void OnPreRender(EventArgs e)
        {
            base.OnPreRender(e);
        }

        protected override void RenderWebPart(HtmlTextWriter output)
        {
            base.RenderWebPart(output);
        }

        #endregion
    }
}

File: trunk/VB/Sample Project/SampleProject/SampleProject.UI/Base/BaseWebPart.vb
Imports System.Web.UI
Imports Microsoft.SharePoint.WebControls
Namespace WebParts
    Public Class BaseWebPart
        Inherits Microsoft.SharePoint.WebPartPages.WebPart
#Region "WebPart Methods"
        Protected Overrides Sub CreateChildControls()
            MyBase.CreateChildControls()
        End Sub


        Protected Overrides Sub OnPreRender(ByVal e As EventArgs)
            MyBase.OnPreRender(e)
        End Sub

        Protected Overrides Sub RenderWebPart(ByVal output As HtmlTextWriter)
            MyBase.RenderWebPart(output)
        End Sub

#End Region
    End Class
End Namespace
